binding/nodejs/test/index.test.ts

```diff
@@ -94,12 +83,12 @@ function testPorcupineDetection(
 
 describe("successful keyword detections", () => {
   it.each(SINGLE_KEYWORD_PARAMETERS)(
-    'testing single keyword for %p with %p', (language, keywords, result, filename) => {
-      testPorcupineDetection(language, keywords, result, filename);
+    'testing single keyword for %p with %p', (language:string, keyword:string, filename:string) => {
```

Comment:
Nitpick: Isn't there usually spaces before the type hints?
```suggestion
    'testing single keyword for %p with %p', (language: string, keyword: string, filename: string) => {
```